native/cocos/editor-support/spine-creator-support/SkeletonDataMgr.cpp

```diff
@@ -46,6 +48,16 @@ class SkeletonDataInfo {
         }
 
         if (atlas) {
+#if CC_USE_SPINE_4_2
+            auto &regions = atlas->getRegions();
+            int size = regions.size();
+            for (int i = 0; i < size; i++) {
```

Comment:
It is better to use range-based for loop

Reply:
spine::Vector can not use range loop